src/components/network-area-diagram-viewer/network-area-diagram-viewer.ts

```diff
@@ -97,6 +97,8 @@ export class NetworkAreaDiagramViewer {
         this.originalWidth = 0;
         this.originalHeight = 0;
         this.dynamicCssRules = customDynamicCssRules ?? DEFAULT_DYNAMIC_CSS_RULES;
+        // if no metadata is available disable node interaction
+        enableNodeInteraction = diagramMetadata == null ? false : enableNodeInteraction;
```

Comment:
We should also disable the `svgDraw.on('mouseover', ...)`. For that, shouldn't we rather give a `hasMetadata` boolean to `init` method?

Reply:
Hover callback disabled if there is no metadata